### ISSUE TITLE
Vulnerability Scanner Question 

### ISSUE THREAD

**Issue by a108a108** [wazuh/wazuh#7392]
Wazuh Version: 4.0.1 running in docker
OS: Ubuntu 16.05 
Install type: packages

Hello,
here is my vulnerability scanner config:
```

<vulnerability-detector>
    <enabled>yes</enabled>
    <interval>7d</interval>
    <ignore_time>5m</ignore_time>
    <run_on_start>no</run_on_start>
```
 

The time of scanning an agent is like 20-30 minutes:
2021/02/07 17:49:52 wazuh-modulesd:vulnerability-detector: INFO: (5450): Analyzing agent '311' vulnerabilities.
2021/02/07 18:15:43 wazuh-modulesd:vulnerability-detector: INFO: (5471): Finished vulnerability assessment for agent '311'

According to the vulnerabilities reported in elsaticsearch it's like a minute.

I have around 250-300 agents and with that pace the scans overlap and it never stops with 7 day interval.  Can I speed up the process ?

**Comment by danimegar** [comment]
Hello @a108a108 

We opened an issue related to a long time that VD takes to scan all the agents and we fixed it in Wazuh 4.0.4. You can check it: https://github.com/wazuh/wazuh/issues/7043.

Maybe that is the problem you have and you only need to update Wazuh to  v4.0.4 and to delete the VD database:
- Stop Manager
- Delete VD database:
`# rm /var/ossec/queue/vulnerabilities/cve.db`
- Start Manager

The VD database will be generated again when VD starts to download and parser the feeds. 

If you prefer to use the version that you are using, you could delete the VD database frequently.

Regards,
Daniel

**Comment by a108a108** [comment]
hi @danimegar,
Thank you, i will update it.
